sugondat-chain/primitives/src/lib.rs

```diff
@@ -5,6 +5,9 @@ use sp_runtime::{
     MultiSignature,
 };
 
+// TODO: probably this could be moved into runtimes/kusama-runtime/src/constants.rs
+pub const MAXIMUM_BLOCK_LENGTH: u32 = 5 * 1024 * 1024;
```

Comment:
Needs docs (btw, this is 5* 1024 * 1024 because it's the maximum allowed PoV size).

Reply:
Previously, `RuntimeBlockLength` was defined like this:
```rust
    pub const RuntimeBlockLength: BlockLength =
        BlockLength::max_with_normal_ratio(5 * 1024 * 1024, NORMAL_DISPATCH_RATIO);
```
I decided to make the value a constant because I needed it inside the `BlobsLengthToFee` logic. However, upon reviewing the code, I realized we could avoid having a fixed `MAXIMUM_BLOCK_LENGTH`. Instead, we could specify `TargetBlockSize` in bytes instead of as a percentage, similar to `TargetBlockFullness`

Therefore, I will remove it!

Reply:
https://research.web3.foundation/Polkadot/overview/token-economics#adjustment-of-fees-over-time

https://github.com/paritytech/polkadot-sdk/blob/9f5221cc2f7f10adfd22b293ceed060617468936/substrate/frame/transaction-payment/src/lib.rs#L97C1-L97C24

the implementation differs from what described and max_limiting_dimension is needed, so `MAXIMUM_BLOCK_LENGTH` should be kept

Reply:
What size is used in Substrate as the default maximum block size? I am 80% sure it's 4 MiB.

Reply:
Both in [substrate node template](https://github.com/paritytech/polkadot-sdk/blob/dcbc36a1c4debf8a4a2714fb5dcf0454dfa31249/substrate/bin/node/runtime/src/lib.rs#L207) and [cumulus node template](https://github.com/paritytech/polkadot-sdk/blob/dcbc36a1c4debf8a4a2714fb5dcf0454dfa31249/cumulus/parachain-template/runtime/src/lib.rs#L253C7-L253C7) is used 5 * 1024 * 1024 as BlockLength